style.css

```diff
@@ -164,6 +169,106 @@ ul {
   justify-content: space-between;
 }
 
+/* -------------POPUP WINDOW------------ */
+
+.popup {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  z-index: 1000;
+  background-color: #c1c7d0;
+}
+
+.popup.active {
+  display: inline-block;
+}
+
+.popup-content {
+  position: fixed;
+  left: 50%;
+  top: 0;
+  transform: translateX(-50%);
+  height: 95%;
+  min-width: 340px;
+  width: 45%;
+  background-color: #fff;
+  transition: top 250ms linear, opacity 200ms linear;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```

Comment:
- [ ] Because `.popup-content `has ` justify-content: space-between;`, on long mobile screens there is a lot of space between image and text 
![image](https://user-images.githubusercontent.com/77038610/146390295-67aa02ab-02d9-4cb1-baaa-3ed5b110cf05.png)

 recommend you to delete this property 👍🏻